src/node-file-trace.ts

```diff
@@ -207,12 +219,15 @@ export class Job {
   async readFile (path: string): Promise<string | Buffer | null> {
     const cached = this.fileCache.get(path);
     if (cached !== undefined) return cached;
+    await this.fileIOQueue.acquire();
     try {
       const source = (await fsReadFile(path)).toString();
       this.fileCache.set(path, source);
+      await this.fileIOQueue.release();
```

Comment:
Nit: Flip these to make it consistent with the others:
```suggestion
      this.fileIOQueue.release();
      this.fileCache.set(path, source);
```